SilKit/source/util/SignalHandler.cpp

```diff
@@ -188,7 +205,27 @@ class SignalMonitor
     int _signalNumber{-1};
 };
 
-void systemHandler(int sigNum)
+static inline void setSignalAction(int sigNum, __sighandler_t action)
+{
+    // Check current signal handler action to see if it's set to SIGNAL IGNORE
+    struct sigaction oldAction;
```

Comment:
```suggestion
    struct sigaction oldAction{};
```
better default initialize variables on stack